cli/tests/pcluster/config/test_validators.py

```diff
@@ -546,6 +547,98 @@ def test_s3_validator(mocker, boto3_stubber, config, num_calls, bucket, expected
     utils.assert_param_validator(mocker, config, expected_message)
 
 
+@pytest.mark.parametrize(
+    "bucket, region, error_code, expected_message, client_error",
+    [
+        (
+            "bucket",
+            "us-east-1",
```

Comment:
I would add another test case for a region that passes the validator and is not `us-east-1`, since `us-east-1` is a special case, we also want to test that other "normal" region can pass the validator.

Reply:
Added